includes/functions.php

```diff
@@ -324,3 +324,42 @@ function register_elementor_wemail_form_action() {
         ->get_modules( 'forms' )
         ->add_form_action( $action->get_name(), $action );
 }
+
+/**
+ * Get host addess from a given url. This excludes 'www.' part also.
+ *
+ * @param  string $url
+ *
```

Comment:
We cloud remove the following two functions: `wemail_get_host_from_url`, `wemail_str_slug`. To get the current site slug, just use `get_option('wemail_site_slug')`

Reply:
Need to remove those functions above.